docs/getting-started/migrate/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Migrate from the legacy Shopify App CLI
+section: getting-started
+redirect_from: "/migrate/"
+---
+
+As of version 0.9.0, Shopify App CLI is installed and managed as a software package, instead of as a git repository. If you’re using a legacy version, you’ll need to perform a one-time migration to keep using the CLI. Follow these steps to remove the legacy version and reinstall as a package.
```

Comment:
The version will definitely change (it is already 0.9.1). I think we could just go with `as of 1.0.0` for the docs as that will be the official one when we release this. Thoughts @mkevinosullivan ?

Reply:
I don't know if we'll go to `1.0.0` immediately. We might reserve that for when we launch the native Windows 10 version.  Likely the launch will be a `0.9.x` and we'll let it sit out there for a couple of iterations before going to `1.0.0`.

If we go with `1.0.0` now, we _might_ need to go to `2.0.0` when the Windows 10 version will be released, because it's uncertain if that will be major enough to break some existing behaviour.  I don't think we want to go from `1.0.0` to `2.0.0` so quickly in succession - better to go from `0.9.x` to `1.0.0`.

Reply:
Yeah, definitely agree, let's leave ourselves plenty of `0.9.x` runway (`0.10.x`, `0.11.x` too, etc)